applications/CompressiblePotentialFlowApplication/custom_elements/embedded_incompressible_potential_flow_element.h

```diff
@@ -0,0 +1,162 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Marc Núñez, based on Inigo Lopez and Riccardo Rossi work
+//
+
+#if !defined(KRATOS_EMBEDDED_INCOMPRESSIBLE_POTENTIAL_FLOW_ELEMENT_H)
+#define KRATOS_EMBEDDED_INCOMPRESSIBLE_POTENTIAL_FLOW_ELEMENT_H
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/element.h"
+#include "includes/kratos_flags.h"
+#include "compressible_potential_flow_application_variables.h"
+#include "utilities/geometry_utilities.h"
+#include "modified_shape_functions/triangle_2d_3_modified_shape_functions.h"
```

Comment:
I think that this one can go to the cpp.

Reply:
I need it in the .h to declare the method:

```
protected:

    ModifiedShapeFunctions::Pointer pGetModifiedShapeFunctions(Vector& rDistances);
```

Reply:
You're right. Now you need it.